File: 125-valid-palindrome/valid-palindrome.js
var isPalindrome = function(s) {
    s = s.toLowerCase().replace(/[^a-zA-Z0-9]/g, "")
    let i = 0
    let j = s.length - 1

    while (i < j) {
        if (s[i] != s[j]) return false
        i += 1
        j -= 1
    }    
    return true
};

File: 125-valid-palindrome/valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        string = ""

        for char in s:
            if char.isdigit() or char.isalpha():
                string += char.lower()
            else:
                continue

        i = 0
        j = len(string) - 1

        while (i < j):
            if (string[i] != string[j]):
                return False
            i += 1
            j -= 1

        return True